src/js/components/FormGroupComponent.jsx

```diff
@@ -1,4 +1,5 @@
 var classNames = require("classnames");
+var dotpathResolve = require("get-object-path");
```

Comment:
Nitpicky little pick: how about `objectPath`? `dotpathResolve` also has the wrong camelCase (should be `dotPathResolve`)